### ISSUE TITLE
pixel size of the miniImagenet in this work

### ISSUE THREAD

**Issue by HHwangqiang** [wyharveychen/CloserLookFewShot#20]
hello, I ran your code on miniImageNet, when I use the dataset with image size 84*84, the results can not reach the  reported accuracy, then I read you code which forms the dataset, I found you make miniImageNet from original image of ImageNet, so I wonder the image size in the ./filelists/miniImagenet  is 84*84 or larger?

**Comment by wyharveychen** [comment]
Hello, note that all my experiment do data augmentation with the RandomSizeCrop so the result would be different from using already cropped 84x84 images.
More specifically, the original size of Imagenet images are 256x256, and I would use RandomSizeCrop to make them 84x84 for Conv4/6 backbone and 224x 224 for ResNet backbone respectively.
For the detail of RandomSizeCrop, please see
https://pytorch.org/docs/stable/_modules/torchvision/transforms/transforms.html#RandomResizedCrop

**Comment by HHwangqiang** [comment]
Oh, thanks for your reply, I change the pixel size of my dataset and now I get the reported result. However, there still a problem confuse me that the result is instable in the training, the test result waving from 70.5 to 73.0, I try to fix the random seed in train.py for both numpy and torch, but it make no sense, then I try to fix the random seed in every .py file, but it does not work too. If you have any advice to make the result stable?

**Comment by wyharveychen** [comment]
Hello, I have to say accuracy in few-shot is indeed very unstable, as its accuracy largely depends on which data you select as training and test data. The only way to make it more stable is to average more results.

**Comment by LinglanZhao** [comment]
Hi, I used conv4 protonet train_aug 5-way 5-shot on miniImagenet but the test accuracy was about 68%.  How can you get the test result from 70.5 to 73.0?  (How did you modify the dataloader?) Thanks a lot! @HHwangqiang
